docs/tutorials/infer_phys_pars.rst

```diff
@@ -306,15 +304,14 @@ curves in :math:`i_\mathrm{p}`--:math:`\Omega_\mathrm{p}` space.
     ii_ccw = (i_p <= 90.*u.deg)
     ii_cw =  (i_p >  90.*u.deg)
 
-    omega_p_stitch1 = np.concatenate((omega_p1[ii_ccw], omega_p2[ii_cw]))
-    omega_p_stitch2 = np.concatenate((omega_p2[ii_ccw], omega_p1[ii_cw]))
+    omega_p = np.concatenate((omega_p[:,ii_ccw], omega_p[::-1,ii_cw]), axis=1)
 
 .. jupyter-execute::
 
     plt.figure(figsize=(7., 6.))
 
-    plt.plot(i_p, omega_p_stitch1, c='C0')
-    plt.plot(i_p, omega_p_stitch2, c='C0')
+    plt.plot(i_p, omega_p[0,:].to(u.deg), c='C0')
```

Comment:
I'd have thought a single plot command would suffice!? (i.e., that it automatically draws the two curves from the 2D array)

Reply:
I thought so too, but when I first tried it didn't work. Turns out I just had to transpose the 2D array.